src/data/de/index.ts

```diff
@@ -45,8 +45,8 @@ export const instanceData = {
       comment: "Kommentar",
       revision: "Bearbeitung",
       thread: "Diskussion",
-      topic: 'Thema',
-      subject: 'Fach',
+      topic: 'Topic',
+      subject: 'Subject'
```

Comment:
hm, this shouldn't change...

Reply:
new strings… I added the translation in crowdin, will be available here soon.